check_borg

```diff
@@ -66,6 +69,9 @@ while getopts ":vhR:C:c:w:" opt; do
 		R)
 			export "BORG_REPO=${OPTARG}"
 			;;
+		P)
+			export "BORG_PREFIX=${OPTARG}"
```

Comment:
2 things:

- I would call the variable `prefix` (since it's not related to borg.
- the export is not necessary, so please don't export it.

Reply:
Agreed! I changed it accordingly.